worlds/sc2/Items.py

```diff
@@ -1465,36 +1497,96 @@ def get_full_item_list():
+    ItemNames.OBSERVER: ItemData(0 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 9, SC2Race.PROTOSS, 
+                 classification=ItemClassification.filler, origin={"wol"},
+                 description="Flying spy. Cloak renders the unit invisible to enemies without detection."),
+    ItemNames.CENTURION: ItemData(1 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 10, SC2Race.PROTOSS, 
+                 classification=ItemClassification.progression, origin={"lotv"},
+                 description="Powerful melee warrior. Has the Shadow Charge and Darkcoil abilities."),
+    ItemNames.SENTINEL: ItemData(2 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 11, SC2Race.PROTOSS, 
+                 classification=ItemClassification.progression, origin={"lotv"},
+                 description="Powerful melee warrior. Has the Charge and Reconstruction abilities."),
+    ItemNames.SUPPLICANT: ItemData(3 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 12, SC2Race.PROTOSS, 
+                 classification=ItemClassification.filler, important_for_filtering=True, origin={"ext"},
+                 description="Powerful melee warrior. Has powerful damage resistant shields."),
+    ItemNames.INSTIGATOR: ItemData(4 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 13, SC2Race.PROTOSS, 
+                 classification=ItemClassification.progression, origin={"ext"},
+                 description="Ranged support strider. Can store multiple Blink charges."),
+    ItemNames.SLAYER: ItemData(5 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 14, SC2Race.PROTOSS, 
+                 classification=ItemClassification.progression, origin={"ext"},
+                 description="Ranged attack strider. Can use the Phase Blink and Phasing Armor abilities."),
+    ItemNames.SENTRY: ItemData(6 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 15, SC2Race.PROTOSS, 
+                 classification=ItemClassification.progression, origin={"lotv"},
+                 description="Robotic support unit can use the Guardian Shield ability and restore the shields of nearby Protoss units."),
+    ItemNames.ENERGIZER: ItemData(7 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 16, SC2Race.PROTOSS, 
+                 classification=ItemClassification.progression, origin={"lotv"},
+                 description="Robotic support unit. Can use the Chrono Beam ability and become stationary to power nearby structures."),
+    ItemNames.HAVOC: ItemData(8 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 17, SC2Race.PROTOSS,
+                 origin={"lotv"}, important_for_filtering=True,
+                 description="Robotic support unit. Can use the Target lock and Force Field abilities and increase the range of nearby Protoss units."),
+    ItemNames.SIGNIFIER: ItemData(9 + SC2LOTV_ITEM_ID_OFFSET, "Unit", 18, SC2Race.PROTOSS, 
+                 classification=ItemClassification.progression, origin={"ext"},
+                 description="Potent cloaked psionic master. Can use the Feedback and Crippling Psionic Storm abilities. Can merge into an Archon."),
```

Comment:
Maybe mention that they're permanently cloaked?

Reply:
fixed

Reply:
Whoops, my eyes skipped right over "cloaked" the first time around. Permanently cloaked still works, though so all good.